internal/server/data/token.go

```diff
@@ -51,6 +52,7 @@ func createJWT(db *gorm.DB, identity *models.Identity, groups []string, expires
 	custom := claims.Custom{
 		Name:   identity.Name,
 		Groups: groups,
+		Nonce:  generate.MathRandom(10),
```

Comment:
This is a math random rather than a crypto random to not drain the entropy on a field that is not used.